src/ucp/wireup/select.c

```diff
@@ -625,8 +637,8 @@ static UCS_F_NOINLINE ucs_status_t ucp_wireup_select_transport(
         /* If a local resource cannot reach any of the remote addresses,
          * generate debug message. */
         if (!is_reachable) {
-            ucs_trace(UCT_TL_RESOURCE_DESC_FMT" : unreachable ",
-                      UCT_TL_RESOURCE_DESC_ARG(resource));
+            ucs_diag(UCT_TL_RESOURCE_DESC_FMT" : unreachable [%s]",
+                     UCT_TL_RESOURCE_DESC_ARG(resource), wireup_info);
```

Comment:
can wireup_info be empty string here?

Reply:
It can't be empty unless we choose to not return info string in some cases (such invalid params).. 
At first I was making sure it's filled in every scenario but you're right that the code doesn't ensure that